libs/api/src/lib/graphql/user/user.queries.ts

```diff
@@ -57,25 +57,159 @@ const createNameFilter = (filter: Partial<UserFilter>): Prisma.UserWhereInput =>
 
   return {}
 }
+const createUserNameFilter = (filter: Partial<UserFilter>): Prisma.UserWhereInput => {
+  const splitedString = filter.text.split(' ')
 
-const createTextFilter = (filter: Partial<UserFilter>): Prisma.UserWhereInput => {
-  if (filter?.text) {
+  if (splitedString.length === 1) {
     return {
       OR: [
         {
-          preferredName: {
```

Comment:
Shouldn't the search still check `preferredName` in addition to first and last?

Reply:
@penk85 ist still there just look at the new file without the changes there you will see that its still in the filter :-)